packages/lambdas/src/document-downloader.ts

```diff
@@ -77,15 +77,19 @@ export const handler = Sentry.AWSLambda.wrapHandler(
       throw new Error(`Config error - CW_ORG_PRIVATE_KEY doesn't exist`);
     }
 
-    const pass = new PassThrough();
+    const { writeStream, promise } = uploadDocumentToS3(
+      fileInfo.fileName,
+      fileInfo.fileLocation,
+      document.mimeType
+    );
 
     let downloadedDocument = "";
 
-    pass.on("data", chunk => {
+    writeStream.on("data", chunk => {
       downloadedDocument += chunk;
     });
 
-    pass.on("end", () => {
+    writeStream.on("end", () => {
       console.log("Finished downloading document");
     });
```

Comment:
I think we don't need that

Reply:
I thought we'd still want it instead of redownloading to the doc from s3 but i can remove. lmk

Reply:
oh, gotcha, this is not being used to dowload, just to process the XML, right?

Reply:
yeah, got it... it makes sense to keep this 👍